DriverLoader.cpp

```diff
@@ -37,6 +37,10 @@
 #define WIN32_LEAN_AND_MEAN
 #define NO_MINMAX
 #include <windows.h>
+#elif defined(OSVR_LINUX)
+#include <dlfcn.h>
+#elif defined(OSVR_MACOSX)
+#include <dlfcn.h>
```

Comment:
Is `dlfcn.h`  how you open a `.dylib` on Mac OS X, or just how you open a `.so`? The terribly confusing part is that I know OS X has both of those... (@godbyk? @d235j?)